README.md

```diff
@@ -171,14 +171,34 @@ before contributing to the project. <!-- TODO: link to guidelines -->
 
 ## Building Multipass
 
-For build instructions, please refer to the following files:
+Please follow the platform-specific build instructions in the files below:
 
-- [BUILD.linux.md](BUILD.linux.md) for Linux
-- [BUILD.macOS.md](BUILD.macOS.md) for macOS
-- [BUILD.windows.md](BUILD.windows.md) for Windows
+- [BUILD.linux.md](./BUILD.linux.md) for Linux  
+- [BUILD.macOS.md](./BUILD.macOS.md) for macOS  
+- [BUILD.windows.md](./BUILD.windows.md) for Windows  
+
+## Important Notices
+
+**Qt version compatibility**  
+Multipass is tested with **Qt 6.2.4**. Newer versions like Qt 6.6+ may cause compatibility issues. Avoid using "latest stable" unless you're certain it's supported.
+
+**ARM64 and cross-compilation**  
+If you're using Apple Silicon (arm64) or cross-compiling, ensure that your `PATH` and `CMAKE_PREFIX_PATH` environment variables point to the correct Qt installation for your system architecture.
+
+**Homebrew**  
+While Multipass can be installed via Homebrew, this method is community-maintained and not officially supported. It is recommended to use the official installer from GitHub.
+
+**Windows / Hyper-V requirement**  
+Hyper-V is required only if you are using the Hyper-V backend. It is not necessary for VirtualBox or WSL2.  
+To check if Hyper-V is enabled, run this in PowerShell:
+
+```powershell
+Get-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V-All
+```
```

Comment:
Multipass doesn't use WSL2... but anyway, this is another paragraph that looks out of place to me. Supported drivers are discussed in our documentation:

- https://documentation.ubuntu.com/multipass/en/latest/explanation/driver/
- https://documentation.ubuntu.com/multipass/en/latest/how-to-guides/install-multipass/

The code does the necessary checks automatically.